docs/templates/modules/hub.md

```diff
@@ -21,6 +21,8 @@ On this page:
 - **storageSku** (string) – Optional. Storage SKU to use. LRS = Lowest cost, ZRS = High availability. Note Standard SKUs are not available for Data Lake gen2 storage. Allowed: `Premium_LRS`, `Premium_ZRS`. Default: `Premium_LRS`.
 - **tags** (object) – Optional. Tags to apply to all resources. We will also add the `cm-resource-parent` tag for improved cost roll-ups in Cost Management.
 - **exportScopes** (array) – Optional. List of scope IDs to create exports for.
+- **exportRetentionInDays** (int) – Optional. Number of days of cost data to retain in the ms-cm-exports container. Default: 0.
+- **ingestionRetentionInMonths** (int) – Optional. Number of months of cost data to retain in the ingestion container. Default: 13.
```

Comment:
@MSBrett I added pipelines for data retention since I don't think we can reuse storage retention for month-based retention. Let me know your thoughts. We could also just round up the number of days in storage retention to avoid extra pipeline run costs. Resolve to ack or comment if you think we should change it.

@tsilvers-ms / @ankur-ms, I'd love your thoughts as well!